features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -310,6 +310,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.modbus/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-neeo" description="NEEO Binding" version="${project.version}">
```

Comment:
What about the org.openhab.io.neeo bundle? Shouldn't you also add another feature for it in the I/O section?